user/models.py

```diff
@@ -3,9 +3,16 @@
 
 
 class User(AbstractUser):
-    is_auto_show = models.BooleanField(default=False)
-    is_customer = models.BooleanField(default=False)
-    is_producer = models.BooleanField(default=False)
+    USERTYPE = (
+        ('AutoShow', 'AutoShow'),
+        ('Customer', 'Customer'),
+        ('Producer', 'Producer'),
+        ('Admin', 'Admin'),
+    )
+    usertype = models.CharField(max_length=50, choices=USERTYPE)
```

Comment:
specify max_length equal to the longest string in choices

Reply:
done)

Reply:
push commit))